### ISSUE TITLE
Add a command for toggling decorations

### ISSUE THREAD

**Issue by brxck** [pokey/cursorless-talon#2]
It would be nice to have something hooked up to `cursorless.toggleDecorations` by default.

**Comment by pokey** [comment]
You mean a voice command? Yeah that's a good idea. Any suggestions? I used to use a parrot noise when I used to toggle them on and off all the time. Fwiw I eventually just stopped noticing them so stopped turning them off but agree it would be good to have the command mapped

**Comment by brxck** [comment]
That's a good question. Might be nice to have a cursorless scope/prefix if you imagine having more of these sort of commands which aren't directly related to editing. 

I've currently got it set up as `marks switch` but I didn't put much thought into that decision/haven't needed to use it much.

**Comment by AndreasArvidsson** [comment]
Good idea. My imagination-less idea:
cursorless (show | hide)

Btw maybe cursorless.toggleDecorations should be enable/disable cursorless so all commands stop as well? Now you can chuck tokens randomly using markers you cant see. Would be nice to have a voice command to actually disable/enable cursorloss.

**Comment by brxck** [comment]
I think it would make sense to switch off cursorless when decorations are hidden. Unless there's a good reason not to?

**Comment by pokey** [comment]
Agreed that we should probably set a mode so that cursorless commands aren't active if cursorless is hidden

Although maybe you'd still want to be able to do things like "pre funk" even if decorations are hidden?

And fwiw, you wouldn't actually be able to chuck tokens you can't see. When you hide the annotations they all get deleted. So talon would recognise the command "chuck air" but cursorless would just throw an exception saying the decoration doesn't exist
